docs/apis/topaz_cli.md

```diff
@@ -933,4 +933,91 @@ Parameter&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Abbr.  | Description
 `-outputarchive` | -oar | A new archive to be created instead of updating the input archive. Default is to update the input archive.
 `-project` | -p | Path to the project folder. If provided, the result and report parameters can be omitted, and their file names will be automatically derived. This parameter has no default.
 `-fileencoding` | -fe | The character encoding to be used for output files. Defaults to UTF-8.
-`-save` | -s | Save output or not. Set to TRUE to write output, or FALSE for no output. When set to FALSE, no files will be created and the archive will not be updated. Defaults to TRUE.
\ No newline at end of file
+`-save` | -s | Save output or not. Set to TRUE to write output, or FALSE for no output. When set to FALSE, no files will be created and the archive will not be updated. Defaults to TRUE.
+
+## Topaz Enterprise Data CLI (TedCLI.bat)
+
+The Topaz Enterprise Data CLI executes TED specifications such ConverterPro, ComparePro, Related Extract, Related Loader and Execution Suites. The specifications will have to be created using the relevant application client available in the File-AID/EX perspective of Topaz and the specifications should be saved and made available in any of the shared repositories. Moreover the shared repository should be configured on a standalone communication manager for using it with TED CLI. Further, to execute the specifications via the TED CLI, standalone execution server needs to be installed if not already available. Please note that the embedded communication manager and execution server that is available with the Topaz TED feature cannot be used for executing TED CLI.
```

Comment:
"TED specifications such ConverterPro..."
including instead of such would be better

Reply:
Made that one plus additional changes to the content.